devsetup/edpm/edpm-play.yaml

```diff
@@ -15,11 +15,11 @@ spec:
           shell: |
             rpm -q git || sudo yum -y install git
             sudo yum -y install python-setuptools python-requests python3-pip
-            git clone https://git.openstack.org/openstack/tripleo-repos
-            pushd tripleo-repos
+            git clone https://github.com/openstack-k8s-operators/repo-setup
+            pushd repo-setup
             sudo python3 setup.py install
             popd
-            sudo /usr/local/bin/tripleo-repos current-tripleo-dev
+            sudo /usr/local/bin/repo-setup current-tripleo-dev
```

Comment:
we can replace current-tripleo-dev with current-podified-dev

Reply:
Agreed, that is the only required change I see here